### ISSUE TITLE
Cannot run training model, stops after just the 1st iteration 

### ISSUE THREAD

**Issue by abstracted26** [monologg/HanBert-NER#1]
sudo python3 main.py --model_type hanbert                   --model_name_or_path HanBert-54kN-torch                  --do_train                   --do_eval
[sudo] password for parallels: 
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   Model name 'HanBert-54kN-torch' not found in model shortcut name list (). Assuming 'HanBert-54kN-torch' is a path, a model identifier, or url to a directory containing tokenizer files.
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   Didn't find file HanBert-54kN-torch/added_tokens.json. We won't load it.
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   Didn't find file HanBert-54kN-torch/special_tokens_map.json. We won't load it.
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   Didn't find file HanBert-54kN-torch/tokenizer_config.json. We won't load it.
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   loading file HanBert-54kN-torch/vocab_54k.txt
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   loading file HanBert-54kN-torch/libmoran4dnlp.so
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   loading file None
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   loading file None
10/25/2020 01:08:22 - INFO - transformers.tokenization_utils -   loading file None
10/25/2020 01:08:23 - INFO - data_loader -   Loading features from cached file ./data/cached_naver-ner_HanBert-54kN-torch_50_test
10/25/2020 01:08:23 - INFO - data_loader -   Loading features from cached file ./data/cached_naver-ner_HanBert-54kN-torch_50_train
10/25/2020 01:08:25 - INFO - transformers.configuration_utils -   loading configuration file HanBert-54kN-torch/config.json
10/25/2020 01:08:25 - INFO - transformers.configuration_utils -   Model config BertConfig {
  "_num_labels": 30,
  "architectures": null,
  "attention_probs_dropout_prob": 0.1,
  "bos_token_id": null,
  "decoder_start_token_id": null,
  "directionality": "bidi",
  "do_sample": false,
  "early_stopping": false,
  "eos_token_id": null,
  "finetuning_task": "naver-ner",
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "id2label": {
    "0": "UNK",
    "1": "O",
    "10": "LOC-B",
    "11": "LOC-I",
    "12": "CVL-B",
    "13": "CVL-I",
    "14": "DAT-B",
    "15": "DAT-I",
    "16": "TIM-B",
    "17": "TIM-I",
    "18": "NUM-B",
    "19": "NUM-I",
    "2": "PER-B",
    "20": "EVT-B",
    "21": "EVT-I",
    "22": "ANM-B",
    "23": "ANM-I",
    "24": "PLT-B",
    "25": "PLT-I",
    "26": "MAT-B",
    "27": "MAT-I",
    "28": "TRM-B",
    "29": "TRM-I",
    "3": "PER-I",
    "4": "FLD-B",
    "5": "FLD-I",
    "6": "AFW-B",
    "7": "AFW-I",
    "8": "ORG-B",
    "9": "ORG-I"
  },
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "is_decoder": false,
  "is_encoder_decoder": false,
  "label2id": {
    "AFW-B": 6,
    "AFW-I": 7,
    "ANM-B": 22,
    "ANM-I": 23,
    "CVL-B": 12,
    "CVL-I": 13,
    "DAT-B": 14,
    "DAT-I": 15,
    "EVT-B": 20,
    "EVT-I": 21,
    "FLD-B": 4,
    "FLD-I": 5,
    "LOC-B": 10,
    "LOC-I": 11,
    "MAT-B": 26,
    "MAT-I": 27,
    "NUM-B": 18,
    "NUM-I": 19,
    "O": 1,
    "ORG-B": 8,
    "ORG-I": 9,
    "PER-B": 2,
    "PER-I": 3,
    "PLT-B": 24,
    "PLT-I": 25,
    "TIM-B": 16,
    "TIM-I": 17,
    "TRM-B": 28,
    "TRM-I": 29,
    "UNK": 0
  },
  "layer_norm_eps": 1e-12,
  "length_penalty": 1.0,
  "max_length": 20,
  "max_position_embeddings": 512,
  "min_length": 0,
  "model_type": "bert",
  "no_repeat_ngram_size": 0,
  "num_attention_heads": 12,
  "num_beams": 1,
  "num_hidden_layers": 12,
  "num_return_sequences": 1,
  "output_attentions": false,
  "output_hidden_states": false,
  "output_past": true,
  "pad_token_id": 0,
  "pooler_fc_size": 768,
  "pooler_num_attention_heads": 12,
  "pooler_num_fc_layers": 3,
  "pooler_size_per_head": 128,
  "pooler_type": "first_token_transform",
  "prefix": null,
  "pruned_heads": {},
  "repetition_penalty": 1.0,
  "task_specific_params": null,
  "temperature": 1.0,
  "top_k": 50,
  "top_p": 1.0,
  "torchscript": false,
  "type_vocab_size": 2,
  "use_bfloat16": false,
  "vocab_size": 54000
}

10/25/2020 01:08:25 - INFO - transformers.modeling_utils -   loading weights file HanBert-54kN-torch/pytorch_model.bin
10/25/2020 01:08:30 - INFO - transformers.modeling_utils -   Weights of BertForTokenClassification not initialized from pretrained model: ['classifier.weight', 'classifier.bias']
10/25/2020 01:08:30 - INFO - transformers.modeling_utils -   Weights from pretrained model not used in BertForTokenClassification: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias']
10/25/2020 01:08:30 - INFO - trainer -   ***** Running training *****
10/25/2020 01:08:30 - INFO - trainer -     Num examples = 81000
10/25/2020 01:08:30 - INFO - trainer -     Num Epochs = 20
10/25/2020 01:08:30 - INFO - trainer -     Total train batch size = 32
10/25/2020 01:08:30 - INFO - trainer -     Gradient Accumulation steps = 1
10/25/2020 01:08:30 - INFO - trainer -     Total optimization steps = 50640
10/25/2020 01:08:30 - INFO - trainer -     Logging steps = 1000
10/25/2020 01:08:30 - INFO - trainer -     Save steps = 1000
Epoch:   0%|                                                 | 0/20 [00:00<?, ?it/sKilledtion:   0%|                                | 1/2532 [00:21<14:50:35, 21.11s/it
